sdk/src/main/java/com/horizen/account/api/rpc/types/TransactionArgs.java

```diff
@@ -153,7 +153,10 @@ public Message toMessage(BigInteger baseFee) throws RpcException {
             }
             // Backfill the legacy gasPrice for EVM execution, unless we're all zeroes
             if (gasFeeCap.bitLength() > 0 || gasTipCap.bitLength() > 0) {
-                effectiveGasPrice = baseFee.add(gasTipCap).min(gasFeeCap);
+                if (baseFee != null) {
```

Comment:
Is there a reason `baseFee` should ever be `null`? I think we should rather throw an `IllegalArgumentException` if it is.

Reply:
It's not possible that baseFee will ever be null. This change was requester from NCC audit.